docs/getting-started.md

```diff
@@ -4,42 +4,40 @@ layout: page.jade
 path: getting-started
 ---
 ## Install
-We provide some ways to use Garden. You can use it by installing from npm, bower or manually (downloading our [latest release](https://github.com/leroy-merlin-br/garden/releases)):
-
-To install using npm
+Installing Garden is pretty straightforward. You can do it via [npm](https://www.npmjs.com/) or [bower](https://bower.io/) by running one of the commands bellow.
 
+##### Using npm
 ```js
 npm install garden
 ```
 
-To install using bower
-
+##### Using bower
 ```js
 bower install garden
 ```
+<br>
```

Comment:
I noticed that the following paragraph was too close to the previous block, so I used this line break since the paragraph is not related at all to the upper block.